wdl-lint/src/v1/import_whitespace.rs

```diff
@@ -0,0 +1,101 @@
+//! A lint rule for preventing whitespace between imports.
+
+use wdl_ast::v1::ImportStatement;
+use wdl_ast::v1::Visitor;
+use wdl_ast::AstNode;
+use wdl_ast::Diagnostic;
+use wdl_ast::Diagnostics;
+use wdl_ast::Span;
+use wdl_ast::SyntaxKind;
+use wdl_ast::ToSpan;
+use wdl_ast::VisitReason;
+
+use super::Rule;
+use crate::util::lines_with_offset;
+use crate::Tag;
+use crate::TagSet;
+
+/// The identifier for the import sort rule.
+const ID: &str = "ImportWhitespace";
+
+/// Creates a bad import whitespace diagnostic.
+fn bad_import_whitespace(span: Span) -> Diagnostic {
+    Diagnostic::note("No blank lines allowed between imports")
```

Comment:
For consistency with other diagnostics:

```suggestion
    Diagnostic::note("blank lines are not allowed between imports")
```